README.md

```diff
@@ -8,6 +8,7 @@ It is officially maintained by the CoreOS team and meant to be a set of introduc
 
 When designing these guides and tools, the following considerations are made:
 
+* You must use a CoreOS version 773.1.0+ for the `kubelet` to be present in the image
```

Comment:
I've put version requirement at the beginning because I've faced situation when I have deployed stable cluster and tried to follow this doc. I've generated keys, set environment variables and at the middle of that doc it tells me that I have to install new CoreOS version. And I have to upgrade/reinstall my cluster. It is quite annoying.